pkg/providers/amifamily/ami_test.go

```diff
@@ -287,7 +287,7 @@ var _ = Describe("AMIProvider", func() {
 		})
 	})
 	Context("AMI Selectors", func() {
-		It("should have default owners and use tags when prefixes aren't set", func() {
```

Comment:
Can we put some comments on why this is safe? E.g. a link to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions and the quote "When you tag public or shared resources, the tags you assign are available only to your AWS account; no other AWS account will have access to those tags" and that this is ok since the AMI we discover via tags here will only be found if the account owner doing the query has tagged it.